powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -117,6 +117,13 @@ public class PowerAuthServiceConfiguration {
     @Min(0)
     private int activationValidityBeforeActive;
 
+    /**
+     * How many milliseconds should the activation cleanup job look to the past.
+     */
+    @Value("${powerauth.service.scheduled.job.activationsCleanup.lookBackInMilliseconds}")
+    @Min(0)
+    private int activationsCleanupLookBackInMilliseconds;
```

Comment:
What about to start using `java.time.Period` and do not use units in the property name?

Reply:
For activation expiration related events, we typically use millseconds explicitly, to provide the required precision.